packages/autorest.typescript/test/smoke/generated/network-resource-manager/samples-dev/firewallPolicyIdpsSignaturesOverridesPatchSample.ts

```diff
@@ -32,7 +32,7 @@ async function patchSignatureOverrides() {
     name: "default",
     type: "Microsoft.Network/firewallPolicies/signatureOverrides",
     id: "/subscriptions/e747cc13-97d4-4a79-b463-42d7f4e558f2/resourceGroups/rg1/providers/Microsoft.Network/firewallPolicies/firewallPolicy/signatureOverrides/default",
-    properties: { signatures: { "2000105": "Off", "2000106": "Deny" } },
+    properties: { signatures: { num2000105: "Off", num2000106: "Deny" } },
```

Comment:
num prefix

Reply:
the definition is record and we should not normalize their keys. this is a bug in sample gen.

```
signatures?: { [propertyName: string]: string };
```